signal.py

```diff
@@ -72,6 +75,21 @@ def __init__(self, signal_file_name, signal_type):
         self.signal_samples = pandas_data_framed_signal.to_numpy()
         self.features = []
 
+    def save_signal_csv(self, file_name):
```

Comment:
Czy nie lepiej jeśli funkcja save_signal_csv będzie w scenarios.py? Bo troszkę mi się nie podoba, że jedno zapisywanie csv jest w jednej klasie, a drugie w drugiej i ta jedna funkcja wywołuje bardzo podobną funckje tylko w innej klasie. Chyba, że są jakieś przeciwskazania, żeby ta funkcja nie była w scenarios.py? Bo ja to widzę dlatego, że to nie jest metoda jakaś do konfiguracji i zawsze, dla każdego scenariusza będzie odpalana.

Reply:
Sugerowałam się tym, że jest to funkcja, która dotyczy konkretnego sygnału, więc wywołuję ją na danym sygnale. Jeśli w dowolnym miejscu będzie sygnał i będzie wymagał zapisania go do csv po przetworzeniu to wciąż można go zapisać, nawet pomocniczo, a nie dla konkretnego scenariusza. Chociaż faktycznie jeśli nie będzie takiej sytuacji i traktujemy tą funkcję jako funkcję pomocniczą do zapisywania scenariusza, to można ją przenieść do Scenerio.

Reply:
Może zróbmy na ten moment to jako funkcję w Scenario bo w tym momencie nie ma powodu, żeby zapisać sygnał w połowie przetwarzania. Co oznacza, ze przerobienie jednego scenariusza = 1 csv z features i 1 csv z sygnałem. Tylko zmieniłbym nazwy funkcji, żeby były bardziej wyjaśniające np save_feature csv i save_signal_csv - tutaj jak już chcesz. A teraz, raz jest save w nazwie, a raz ma write, a robią praktycznie to samo xd

Reply:
Dobra, ma to sens, zgadzam się